Compilation failure in ApiUtil class for openapi

Compilation failure for the below combination

```
/tmp/webflux-gradle-session-h2mem-es/build/openapi/src/main/java/tech/jhipster/sample/web/api/ApiUtil.java:5: error: package javax.servlet.http does not exist
import javax.servlet.http.HttpServletResponse;
```

##### **JHipster Version(s)**

main branch

Java verion: openjdk 17.0.8 2023-07-18

##### **JHipster configuration**

```
{
  "generator-jhipster": {
    "applicationType": "monolith",
    "authenticationType": "session",
    "baseName": "sampleWebfluxH2mem",
    "buildTool": "gradle",
    "cacheProvider": "no",
    "clientFramework": "angular",
    "clientPackageManager": "npm",
    "clientTheme": "none",
    "clientThemeVariant": "",
    "creationTimestamp": 1596513272473,
    "databaseType": "sql",
    "devDatabaseType": "h2Memory",
    "devServerPort": 4200,
    "dtoSuffix": "DTO",
    "enableGradleEnterprise": false,
    "enableHibernateCache": false,
    "enableSwaggerCodegen": true,
    "enableTranslation": true,
    "entities": [],
    "entitySuffix": "",
    "fakerSeed": "1",
    "jhiPrefix": "jhi",
    "jhipsterVersion": "8.0.0-beta.2",
    "languages": ["en", "fr"],
    "messageBroker": "no",
    "nativeLanguage": "en",
    "packageFolder": "tech/jhipster/sample",
    "packageName": "tech.jhipster.sample",
    "pages": [],
    "prodDatabaseType": "postgresql",
    "reactive": false,
    "rememberMeKey": "a5e93fdeb16e2ee2dc4a629b5dbdabb30f968e418dfc0483c53afdc695cfac96d06cf5c581cbefb93e3aaa241880857fcafe",
    "searchEngine": "elasticsearch",
    "serverPort": "8080",
    "serviceDiscoveryType": "no",
    "skipCheckLengthOfIdentifier": false,
    "skipClient": false,
    "skipFakeData": false,
    "skipUserManagement": false,
    "testFrameworks": ["gatling", "cypress"],
    "websocket": "no",
    "withAdminUi": true
  }
}
```

--- a/generators/server/templates/gradle/swagger.gradle.ejs
+++ b/generators/server/templates/gradle/swagger.gradle.ejs
@@ -31,7 +31,7 @@ openApiGenerate {
     apiFilesConstrainedTo = [""]
     modelFilesConstrainedTo = [""]
     supportingFilesConstrainedTo = ["ApiUtil.java"]
-    configOptions = [delegatePattern: "true", title: "<%= dasherizedBaseName %>"<% if (reactive) {  %>, reactive: "true"<% } %>]
+    configOptions = [delegatePattern: "true", title: "<%= dasherizedBaseName %>"<% if (reactive) {  %>, reactive: "true"<% } %>, useSpringBoot3: "true"]
     validateSpec = true
     importMappings = [Problem:"org.springframework.http.ProblemDetail"]
 }

